src/test/ui/inference/inference_unstable.rs

```diff
@@ -15,5 +15,7 @@ use inference_unstable_itertools::IpuItertools;
 fn main() {
     assert_eq!('x'.ipu_flatten(), 1);
     //~^ WARN a method with this name may be added to the standard library in the future
-    //~^^ WARN once this method is added to the standard library, the ambiguity may cause an error
+    //~| WARN a method with this name may be added to the standard library in the future
+    //~| WARN once this method is added to the standard library, the ambiguity may cause an error
```

Comment:
These duplicated error messages seem undesirable.

Reply:
They get deduplicated before being presented to end users, they are duplicated in test files because we pass a special flag to _detect_ when this happens. This effect should be fine.

Reply:
The duplication (whether visible or not) is no longer present after the last commit :)